src/PersonObjects/Work/Work.ts

```diff
@@ -0,0 +1,21 @@
+import { IPlayer } from "../IPlayer";
+
+export enum WorkType {
+  CreateProgram = 0,
```

Comment:
shouldn't we keep it as string enums, more reusability for user facing stuff? or reason to move to ints?